sdk/communication/CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing Guide
```

Comment:
I think we should link to the main contribution guide here as well and also in different sections, point to the correct base document